examples/oak/README.md

```diff
@@ -0,0 +1,28 @@
+# Amiga Brain Camera example
```

Comment:
Do we want these instructions instead on the website?

Reply:
This are example depending. Possibly we need to start an examples section in docs